gnomad_qc/v4/create_release/create_release_sites_ht_exomes.py

```diff
@@ -38,55 +38,53 @@
 logger = logging.getLogger("create_release_ht")
 logger.setLevel(logging.INFO)
 
-# Remove InbreedingCoeff from allele-specific fields and SOR from SITE
-# (processed separately from other fields)
+# Remove InbreedingCoeff from allele-specific fields because it is processed separately
+# from the other fields.
 AS_FIELDS = deepcopy(AS_FIELDS)
 AS_FIELDS.remove("InbreedingCoeff")
 SITE_FIELDS = deepcopy(SITE_FIELDS)
-SITE_FIELDS.remove("SOR")
 TABLES_FOR_RELEASE = [
     "dbsnp",
-    # "filters",
+    "filters",
     "freq",
-    # "info",
+    "info",
     "region_flags",
     "in_silico",
     "vep",
 ]
-
-INSILICO_PREDICTORS = ["spliceai", "pangolin", "revel"]  # "cadd", "phylop"
+INSILICO_PREDICTORS = ["spliceai", "pangolin", "revel", "cadd"]  # , "phylop"
 
 
-# Putting this in a function so that it is not evaluated until the script is run.
+# Config is added as a function, so it is not evaluated until the function is called.
 def get_config(release_exists: bool = False):
     """
     Get configuration dictionary.
 
     Format:
         '<Name of dataset>': {
-                'path': 'gs://path/to/hailtable.ht',
+                'path': 'gs://path/to/hail_table.ht',
                 'select': '<Optional list of fields to select or dict of new field name to location of old fieldin the dataset.>',
                 'field_name': '<Optional name of root annotation in combined dataset, defaults to name of dataset.>',
                 'custom_select': '<Optional function name of custom select function that is needed for more advanced logic>',
-                'select_globals': '<Optional list of globals to select or dict of new global field name to old global field name. If not specified, all globals are selected.>
+                'select_globals': '<Optional list of globals to select or dict of new global field name to old global field name. If not specified, all globals are selected.>'
             },
 
     :param release_exists: Whether the release HT already exists.
-    :return: dict of datasets configs.
+    :return: Dict of datasets configs.
     """
     config = {
         "dbsnp": {
             "ht": dbsnp.ht(),
             "path": dbsnp.path,
             "select": ["rsid"],
         },
-        # "filters": {
-        #     "ht": final_filter().ht(),
-        #     "path": final_filter().path,
-        #     "select": ["filters", "vqsr"],
-        #     "custom_select": custom_filters_select,
-        #     "select_globals": ["filtering_model", "inbreeding_coeff_cutoff"],
-        # },
+        "filters": {
+            "ht": final_filter().ht(),
+            "path": final_filter().path,
+            "select": ["filters"],
+            "custom_select": custom_filters_select,
+            "select_globals": ["filtering_model", "inbreeding_coeff_cutoff"],
```

Comment:
filtering_model contains 'model_id' in it, but I think that is only useful internally so we should drop it from the release HT.